ssl-config-generator/index.html

```diff
@@ -214,6 +215,17 @@
                     cipherSuites: cipherSuites[data.securityProfile],
                     clientList: clientList[data.securityProfile]
                 });
+
+                if (isOpenSSLSemVer(data.opensslVersion, "<1.0.1") && data.securityProfile == "modern") {
+                	$("#security-profile-list input#intermediate").prop( "checked", true );
+                	data.securityProfile = "intermediate";
+                	$("#security-profile-list input#modern").prop("disabled", true);
+                	$("#security-profile-list label[for=modern]").attr("title", "Modern profile is not possible with OpenSSL <1.0.1");
```

Comment:
Could the message indicate `why` modern is not possible? Like: is tlsv1.{1,2} support missing? no cipher overlap? etc...